rebasehelper/application.py

```diff
@@ -581,7 +581,8 @@ def build_packages(self):
                     msg = 'Building {} RPM packages failed; see {} for more information'.format(version, e.logfile)
                 raise RebaseHelperError(msg, logfiles=builder.get_logs().get('logs'))
 
-            except Exception:
+            except Exception as e:
+                logger.error(e)
```

Comment:
Wouldn't it be better to append the exception message to the RebaseHelperError which is being raised anyway?

Reply:
Probably yes. But isn't it better to have the errors in logs too for consistency?

Reply:
Oh I see it already. You're right, your approach is sufficient.

Reply:
Yes, of course. But that's already taken care of: https://github.com/rebase-helper/rebase-helper/blob/master/rebasehelper/cli.py#L282-L286